tanner/api/api.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, redis_client):
     async def return_snares(self):
         query_res = []
         try:
-            query_res = await self.redis_client.smembers('snare_ids', encoding='utf-8')
+            query_res = await self.redis_client.smembers('key:snare_ids', encoding='utf-8')
```

Comment:
The previous one is working fine for me. Did it give some error or return the wrong results?

Reply:
I tried it without `key:` and error I got - `aioredis.errors.ReplyError: WRONGTYPE Operation against a key holding the wrong kind of value`. Also, while testing using `redis_client.sadd()` gave me same error without using `key:`.

Reply:
Wait same thing I tried [here](https://github.com/mushorg/tanner/blob/d7b6c4b3bb0497941d12b4747fae196f5accd604/tanner/api/api.py#L45) but it works both ways. Let me check this again.

Reply:
Nope `smembers` not working.